### PR TITLE
alsa-info: lots of mostly syntactic changes

### PR DESCRIPTION
**Pull request by E5ten**
- remove all bashisms and change shebang to #!/bin/sh
- replace use of which with command -v for portability
- remove awk dependency
- general cosmetic consistency:
    - indentation
    - always put then or do on the same line as if, for, or while
- quote much more, especially when the variable refers to a file
- replace many uses of >> $FILE with command grouping and single
  redirects of the command groups
- simplify text processing pipelines, mostly by turning them into
  a single sed command

### CODE REVIEW

File: alsa-info/alsa-info.sh [hunk 1]
```diff
@@ -804,163 +807,156 @@ if [ "$UPLOAD" = "ask" ]; then
...
+		else
+			FINAL_URL=$(grep "SUCCESS:" "$TEMPDIR/wget.tmp" | sed -n 's/.*\:\([0-9]\+\).*/http:\/\/pastebin.ca\/\1/p')
 	fi
 fi
 
-# Output the URL of the uploaded file.	
+# Output the URL of the uploaded file.
 echo "Your ALSA information is located at $FINAL_URL"
 echo "Please inform the person helping you."
 echo ""
```
Comment (dpward) [inline alsa-info/alsa-info.sh]: These lines need an extra indent.

Comment (dpward) [inline alsa-info/alsa-info.sh]: These lines need an extra indent.

Comment (E5ten) [reply alsa-info/alsa-info.sh]: Thanks for catching that

**Comment by takaswie** [comment]
Hi,

Thanks for the patch. Although I prefer it, the patch includes the batch of changes. I thinks it better to split the changes into the series of small commit according to each of item which you enumerate in PR comment. In short, the patch is not suitable just for review process, unfortunately.

Thanks.

**Comment by perexg** [comment]
Please, split your changes and rebase to the current alsa-info.sh .

**Comment by perexg** [comment]
No more work on this. Closing.

**Pull request closed without merging**